postgis/postgis_before_upgrade.sql

```diff
@@ -173,3 +173,9 @@ BEGIN
 END;
 $$;
 
+-- FUNCTION ST_AsGeoJson added `id_column` optional argument in 3.5.0.
+SELECT _postgis_drop_function_by_identity
+	(
+	'ST_AsGeoJson',
+	'r record, geom_column text, maxdecimaldigits integer, pretty_bool bool'
```

Comment:
'r record, geom_column text, maxdecimaldigits integer, pretty_bool boolean'

change bool to boolean.  It's not dropping cause the postgresql canonical signature standardizes on boolean

Reply:
Fixed.